src/_gettsim/transfers/rente.py

```diff
@@ -160,7 +170,43 @@ def entgeltp_update(entgeltp: float, entgeltp_update_lohn: float) -> float:
     # Note: We might need some interaction between the two
     # ways to accumulate earnings points (e.g., how to
     # determine what constitutes a 'care period')
-    out = entgeltp + entgeltp_update_lohn
+    if wohnort_ost:
+        out = entgeltp_west
+    else:
+        out = entgeltp_west + entgeltp_update_lohn
+    return out
+
+
+def entgeltp_ost_update(
+    wohnort_ost: bool, entgeltp_ost: float, entgeltp_update_lohn: float
+) -> float:
+    """Update eastern earning points.
+
+    Given earnings, social insurance rules, average
+    earnings in a particular year and potentially other
+    variables (e.g., benefits for raising children,
+    informal care), return the new earnings points.
+
+    Parameters
+    ----------
+    wohnort_ost
+        See basic input variable :ref:`wohnort_ost <wohnort_ost>`.
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>`.
+    entgeltp_update_lohn
+        See :func:`entgeltp_update_lohn`.
+
+    Returns
+    -------
+
+    """
+    # Note: We might need some interaction between the two
```

Comment:
I don't follow this comment. 

In general it is better to open an issue than to leave comments in the code; the latter just gets lost.

Reply:
Me neither to be honest. It was already there in the original version of `entgeltp_update`. According to git blame it was written by @MaxBlesch

Reply:
Ah, I guess that's what I meant with "It just gets lost". :laughing:

Just delete then, @MaxBlesch can always open an issue in case it was some deeper insight :wink: